egs/zeroth_korean/s5/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,277 @@
+#!/bin/bash
+
```

Comment:
These recipes should have results at the top in a comment.  Please create a script like compare_wer.sh that prints out the results and diagnostics; you can copy and modify it from another setup.